src/Util/Message.tcc

```diff
@@ -9,16 +9,15 @@
 
 #include "Message.h"
 
-struct message_parsing_exception : std::runtime_error {
+struct parsing_exception : std::runtime_error {
```

Comment:
I would keep the original name -- it's less ambiguous. This struct is available to any file that imports `Message.h`.